extensions/cornerstone/src/services/ViewportService/CornerstoneViewportService.ts

```diff
@@ -181,6 +182,16 @@ class CornerstoneViewportService
     if (properties) viewport.setProperties(properties);
     const camera = presentations?.positionPresentation?.camera;
     if (camera) viewport.setCamera(camera);
+    if(presentations?.positionPresentation?.slabThickness && presentations.positionPresentation.viewportType === 'volume') {
+      const slabThickness = presentations?.positionPresentation?.slabThickness;
```

Comment:
We already have check in if condition, we can directly access slabthickness property no need for ?.

Better, we should access property only once
let SlabThickness = presentations?.positionPresentation?.slabThickness
if(SlabThickness){
    if (slabThickness <= MINIMUM_SLAB_THICKNESS) {
        viewport.setBlendMode(0); //, actorUIDs, immediate);
        viewport.setSlabThickness(MINIMUM_SLAB_THICKNESS); //, actorUIDs);
      } else {
        viewport.setBlendMode(1); //, actorUIDs, immediate);
        viewport.setSlabThickness(slabThickness); //, actorUIDs);
      }
}

Reply:
Addressed in the latest commit.